eZ Platform/Admin UI/REQUIREMENTS.md

```diff
@@ -0,0 +1,45 @@
+# v2 Hybrid Prototype Admin UI Requirements
+## https://github.com/ezsystems/specs/issues/4
+
+# Objective
+"As an eZ Engineer, I want...
+
+To create a functioning, experimental combination of Symfony3, eZ Platform, and
+a mix of existing elements (in YUI) and rewritten elements (in a new framework),
+for the purpose of research and development (prototype), so that I can get a
+better picture of how such a hybrid approach may work."
+
+Underlying main goals are:
+- by adopting Symfony 3, we will provide developers, and especially the one who are experienced with eZ Publish or Symfony, substantially faster and easier way to develop, extend and customize eZ Platform.
+- by insuring an hybrid architecture that reuses existing elements relying on YUI, we will speed up availability of 2.x by not having to redevelop all features. Candidates for reuse are UDW, Sub-items view, Content view. If this is not possible, expectations and plans for 2.x will have to be be re-adjusted.
```

Comment:
@dpobel I am in general looking to see if we can run existing part of the content view part without having to recode them, and having them work with the new u.i. architecture: 
- view tabs
- details tabs
- versions tabs
- location tabs
- related content tabs
- sub-items view widget
At this point of the exploration, I'm not sure if we have to be more specific but exploring if this could be reused taking one or two example to validate the idea. 
For instance, make sure that we can, from the new routing, trigger the display of those parts.

Down the road, I think it would be really great if we can reuse most of the View screens as they are functional for most of it.

This would mean that at first we could concentrate on the action bar, the discovery bar and the Edit views.

Is that more clear?